app/services/request_create_service.rb

```diff
@@ -8,9 +10,10 @@ def initialize(workflow_id)
   def create(options)
     stages = workflow.group_refs.collect do |group_ref|
       Stage.new(
-        :group_ref => group_ref,
-        :state     => Stage::PENDING_STATE,
-        :decision  => Stage::UNDECIDED_STATUS,
+        :group_ref         => group_ref,
+        :state             => Stage::PENDING_STATE,
+        :decision          => Stage::UNDECIDED_STATUS,
+        :random_access_key => SecureRandom.hex(16)
```

Comment:
I realize `hex` is safe for our needs, but did you consider any of the other options.  

Other services that I've seen use this appear to be using `SecureRandom.urlsafe_base64` and I could see extending beyond the default 16-chars, but I think we can adjust that almost anytime.

Reply:
Agree. We could adjust this anytime as needed.